### PR TITLE
Update the field description for ci_job

### PR DESCRIPTION
**Pull request by LingMQ**
Changed some field name and added some new field.

### CODE REVIEW

File: namespaces/ci_job.yml [hunk 1]
```diff
@@ -16,12 +16,12 @@ namespace:
     description: >
       Name of the CI job.
 
-  - name: build_id
+  - name: number
```
Comment (richm) [inline namespaces/ci_job.yml:19]: `build_id` seems a lot more descriptive than `number` - is this necessary to change?

File: namespaces/ci_job.yml [hunk 2]
```diff
@@ -16,12 +16,12 @@ namespace:
...
     type: integer
     description: >
       Build ID of the CI job.
 
-  - name: build_timestamp
+  - name: timestamp
```
Comment (richm) [inline namespaces/ci_job.yml]: I know this is not a new field, but how is this field different than the top level `@timestamp` field?  Can we use the top level `@timestamp` field instead and get rid of this field?

File: namespaces/ci_job.yml [hunk 3]
```diff
@@ -47,6 +47,21 @@ namespace:
     description: >
       Full URL to the CI job on the CI master.
 
+  - name: node_name
```
Comment (richm) [inline namespaces/ci_job.yml:50]: How is this different than the top level `hostname` field?

File: namespaces/ci_job.yml [hunk 4]
```diff
@@ -47,6 +47,21 @@ namespace:
...
+    type: string
+    description: >
+      The Jenkins node that ran the CI job.
+
+  - name: node_description
+    type: string
+    description: >
+      The description of Jenkins node.
+
+  - name: host_name
```
Comment (richm) [inline namespaces/ci_job.yml]: I don't think this field is named correctly.

File: namespaces/ci_job.yml (hunk 1 above)
Comment (t0ffel) [reply namespaces/ci_job.yml:19]: `build_id` has different meaning in Jenkins, we want `number` here to make it unique and integer.

File: namespaces/ci_job.yml [hunk 5]
```diff
@@ -16,12 +16,12 @@ namespace:
     description: >
       Name of the CI job.
 
-  - name: build_id
+  - name: number
     type: integer
     description: >
       Build ID of the CI job.
```
Comment (t0ffel) [inline namespaces/ci_job.yml]: please update the description as well

File: namespaces/ci_job.yml (hunk 2 above)
Comment (LingMQ) [reply namespaces/ci_job.yml]: this field shows the time for ci job build. While the @timestamp at the top level could show a different time. For example, if the job has the test cases, the @timestamp will be the time for running testcase.

File: namespaces/ci_job.yml (hunk 3 above)
Comment (LingMQ) [reply namespaces/ci_job.yml:50]: the hostname is the Jenkins master. While the node_name shows which node of Jenkins master executes the job.

File: namespaces/ci_job.yml (hunk 2 above)
Comment (richm) [reply namespaces/ci_job.yml]: the top level `@timestamp` is the time that the job started and the `ci_job.timestamp` field is the time at which the build portion of the job started?

File: namespaces/ci_job.yml (hunk 4 above)
Comment (LingMQ) [reply namespaces/ci_job.yml]: This field has been updated to be "hostname"

File: namespaces/ci_job.yml (hunk 5 above)
Comment (LingMQ) [reply namespaces/ci_job.yml]: the description has been updated

File: namespaces/ci_job.yml (hunk 2 above)
Comment (t0ffel) [reply namespaces/ci_job.yml]: `@timestamp` may refer to a more precise timing of the specific record. For example xunit test result might have a timestamp, in that case test's timestamp will get elevated to the top level `@timestamp`, while `ci_job.timestamp` will be preserved for the time of Jenkins build.

**Review by richm — APPROVED** [review]

**Pull request merged**